mods/cnc/languages/lua/en.ftl

```diff
@@ -134,29 +134,24 @@ secure-second-landing-zone = Secure the second landing zone.
 
 ## nod10a
 kill-gdi-scientist = Kill the GDI scientist.
-destroy-tech-center = Destroy the GDI R&D center.
+destroy-tech-center = Destroy the GDI Research Base.
 
 ## nod10b
 destroy-capture-warfactory = Destroy or capture the Weapons Factory.
-destroy-mammoth-tanks = Destroy the Mammoth tanks in the R&D base.
+destroy-mammoth-tanks = Destroy the Mammoth tanks in the Research Base.
 keep-commando-alive = Keep your Commando alive.
 
 ## eviction-notice
-take-civilians-money-crates = Find all the civilians' money.
-    They won't need it anymore.
-quickly-destroy-ion-cannon = Disable GDI Ion Cannon before
-    it fires two times.
-nod-soldier = Nod Soldier
+take-civilians-money-crates = Find all the civilians' money; they won't be needing it anymore.
+quickly-destroy-ion-cannon = Disable the GDI Ion Cannon before it fires twice.
+nod-soldier = Nod Soldier.
 civilians-running = Hey, those civilians... where are they going?
-destroy-ion-cannon-advice = The GDI are preparing their ion cannon. Don't let them get used to it.
-village-destruction-warning = Be careful, commander. The GDI won't stand still while we burn the entire village.
+destroy-ion-cannon-advice = The GDI is preparing their Ion Cannon. Don't let them get comfortable using it.
+village-destruction-warning = Be careful, commander. The GDI won't stay idle while we burn the entire village.
 
 ## twist-of-fate
-clear-path = Repel the ambush and clear the way
-    for our MCV.
-recover-old-base = Capture the Construction Yard in our recon
-    post to regain control.
-air-strikes-intel-report = Nod airstrikes are being directed by a Communications Center, located northwest. We may gain useful information from its capture.
-capture-nod-communications-center = Capture the Nod Communications Center to
-    the northwest.
-communications-center-captured-sams-located = Our engineers located Nod SAM sites. They also disarmed an unusual trap on the Construction Yard to the south.
+clear-path = Repel the ambush and clear the way for our MCV.
+recover-old-base = Capture the Construction Yard at our recon post to regain control.
+air-strikes-intel-report = Nod airstrikes are being directed by a Communications Center to the northwest. We may gain useful information from capturing it.
```

Comment:
"from capturing it" doesn't sound right to me. "from its capture" is fine IMO, otherwise "by capturing it"?